src/components/Image/Image.scss

```diff
@@ -0,0 +1,47 @@
+@import 'src/base-styles/dependencies';
+
+// figure {
+//   margin: 0;
+// }
+
+/* Tamanho da caixa da foto para a página onde temos diversas fotos de produto */
```

Comment:
Chequei todas as classes, os tamanhos estão corretos e de acordo com especificado no figma, boa escolha de nome das classes e muito legal já ter implementado os breakpoints, acrescentaria somente alguns pontos em termos de hierarquia e agrupamento das classes no sass:

-  figure e image no início da folha, uma vez que estão setando estilos indpendente das outras classes;

- renomearia as classes referenciando product como (BLOCO) e  (--cart, --navigation e --detail) como elementos (__cart, __navigation e __detail) ao invés de modificadores, assim evitamos repetição  agrupando os elementos no sass com o uso do & comercial:

```scss
 figure {
   margin: 0;
 }

 img {
   max-width: 100%;
   cursor: pointer;
 }
 
 .product {
   &__navigation {
   width: 40%;
   }

   &__detail {
    width: 100%;
    position: absolute;
    top: 60px;
   }

   &__cart {
    width: 30%;
   }
 }

```

Reply:
As imagens vem da API mesmo. Podemos deletar esse assets se não for utilizado com outra coisa depois. 
Quanto as mudanças beleza. Valeu! Esqueço das adaptações possíveis com o sass.